web/src/components/Trace/__tests__/Trace.test.tsx

```diff
@@ -1,22 +1,101 @@
 import {render, waitFor} from '@testing-library/react';
 import {ReactFlowProvider} from 'react-flow-renderer';
-import TestMock from '../../../models/__mocks__/Test.mock';
-import TestRunMock from '../../../models/__mocks__/TestRun.mock';
 import {ReduxWrapperProvider} from '../../../redux/ReduxWrapperProvider';
 import Trace from '../Trace';
+import {SemanticGroupNames} from '../../../constants/SemanticGroupNames.constants';
+import {TSpan} from '../../../types/Span.types';
+import {TTestRun} from '../../../types/TestRun.types';
+import {TTest} from '../../../types/Test.types';
+import {HTTP_METHOD} from '../../../constants/Common.constants';
+
+const spanId = '4938u43';
```

Comment:
Consider using the mock factories, you can overwrite the different values if you want